src/atoms/buttons/types.ts

```diff
@@ -15,7 +15,6 @@ export interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElemen
   | 'special_extra_tiny_subtle'
   borderRadius?: string,
   disabled?: boolean
-  onClick: () => void;
```

Comment:
onClick already comes from the ButtonHTMLAttributes<HTMLButtonElement> and has all the types